test/controllers/sessions_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class SessionsControllerTest < ActionDispatch::IntegrationTest
+  test "should get new" do
+    get sessions_new_url
```

Comment:
Not a blocker (NAB): We generally use Rspec for tests in ruby. It's worthwhile learning it by the way, as it's probably the most commonly used testing framework for Ruby